homeassistant/components/rituals_perfume_genie/select.py

```diff
@@ -12,6 +18,37 @@
 from .entity import DiffuserEntity
 
 
+@dataclass
+class RitualsEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    current_fn: Callable[[Diffuser], str]
+    select_fn: Callable[[Diffuser, str], Awaitable[Any]]
```

Comment:
Why not use `None` here?
```suggestion
    select_fn: Callable[[Diffuser, str], Awaitable[None]]
```

Reply:
No idea, simply because of habit, I guess. Removed it.